src/video/camerasource.cpp

```diff
@@ -65,12 +65,12 @@ void CameraSource::open()
     open(CameraDevice::getDefaultDeviceName());
 }
 
-void CameraSource::open(const QString deviceName)
+void CameraSource::open(const QString& deviceName)
```

Comment:
We have const. It means no difference between copy and reference. But send by reference more effectively then full copy of string